src/dnx.clr.managed/DomainManager.cs

```diff
@@ -36,12 +37,34 @@ public override void InitializeNewDomain(AppDomainSetup appDomainInfo)
             Environment.SetEnvironmentVariable(EnvironmentNames.AppBase, _info.ApplicationBase);
         }
 
-        var version = DetermineDnxVersion(_info.ApplicationBase);
+        appDomainInfo.ApplicationBase = Environment.GetEnvironmentVariable(EnvironmentNames.DefaultLib);
+        appDomainInfo.TargetFrameworkName = DetermineAppDomainTargetFramework();
+    }
+
+    private string DetermineAppDomainTargetFramework()
+    {
+        // Check if the native bootstrapper gave us a framework name
+        string frameworkName = Environment.GetEnvironmentVariable(EnvironmentNames.Framework);
```

Comment:
/cc @moozzyk @BrennanConroy We need to clean up the env variable thing. This is added to the cruft :smile: